### PR TITLE
fix(pkg/driverbuilder): multiple small fixes to local builder.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug
/kind cleanup

**Any specific area of the project related to this PR?**

/area pkg

**What this PR does / why we need it**:

This PR contains small fixes on top of local builder.
Also, support bumping `dkms log file` if dkms build fails.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

NOTE: this PR bumps falcoctl to latest main, to take advantage of https://github.com/falcosecurity/falcoctl/pull/550.
That is needed to avoid importing `pkg/options` package in driverkit that would lead to an import cycle when later driverkit gets imported in falcoctl.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
NOTE: this is `wip` because i need to fix a small thing on falcoctl before proceeding.

File: pkg/driverbuilder/builder/templates/local.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # looking for it in a bunch of ways. Convenient when running Falco inside
 # a container or in other weird environments.
 #
-set -xeo pipefail
+set -xeuo pipefail
```
Comment (FedeDP) [inline pkg/driverbuilder/builder/templates/local.sh:23]: Same as other tempaltes.

File: pkg/driverbuilder/local.go [hunk 2]
```diff
@@ -166,6 +169,12 @@ func (lbp *LocalBuildProcessor) Start(b *builder.Build) error {
 
 	for _, gcc := range gccs {
 		vv.GccPath = gcc
+		if c.ModuleFilePath != "" {
```
Comment (FedeDP) [inline pkg/driverbuilder/local.go:172]: Added some debug info.

File: pkg/driverbuilder/local.go [hunk 3]
```diff
@@ -181,59 +190,58 @@ func (lbp *LocalBuildProcessor) Start(b *builder.Build) error {
...
-			} else {
-				// print the output of the subprocess line by line
-				scanner := bufio.NewScanner(stdout)
-				for scanner.Scan() {
-					m := scanner.Text()
-					fmt.Println(m)
-				}
-				err = cmd.Wait()
-			}
+		out, err := cmd.CombinedOutput()
```
Comment (FedeDP) [inline pkg/driverbuilder/local.go:193]: No need to pipe and scan stdout/stderr: we are always behind a stylish spinner now, therefore we will never print these lines in real time.

**Review by EXONER4TED — APPROVED** [review]
looks good to me 🙂

**Pull request merged**